cicd-scripts/setup-e2e-tests.sh

```diff
@@ -139,6 +139,7 @@ deploy_hub_spoke_core() {
     latest_release_branch=$(git ls-remote --heads https://github.com/open-cluster-management/registration-operator.git release\* | tail -1 | cut -f 2 | cut -d '/' -f 3)
     git clone --depth 1 -b ${latest_release_branch} https://github.com/open-cluster-management/registration-operator.git && cd registration-operator
 
+    export HUB_KUBECONFIG=${KUBECONFIG}
```

Comment:
where to use this variable?

Reply:
it is used to fix the registration-operator setup issue. Thanks for the quick fix.